src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/21.diff.sql

```diff
@@ -0,0 +1,39 @@
+-- STORED PROCEDURE
+--     RemovePartitionFromResourceChanges
```

Comment:
Since our codebase is backward compatible, the approach that we take to update the existing stored proc from previous version, is by adding incrementing version to it(for ex- upsertResource Stored proc has been updated in few migrations) and then add logic in the code to use the relevant store proc based on the schema version.

Reply:
Renamed the stored procedure to RemovePartitionFromResourceChanges_2.